src/configuration/tree.yaml

```diff
@@ -159,17 +177,50 @@ root:
           - label: "Mi piacerebbe lavorare su SDK e voglio supportare SPID"
             question: "Quanto conosci di SPID?"
             content:
-              - label: "Vai al portale sviluppatore"
-                content: |
-                  <https://developers.italia.it/it/spid>
 
               - label: "Scopri l'introduzione ai concetti principali"
                 content: |
                   <https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/single-sign-on.html>
 
+              - label: "Accedi al portale dello sviluppatore"
+                content: |
+                  Qui trovi le risorse di esempio e i progetti OpenSource dedicati a SPID, oltre alla documentazione tecnica e altre risorse:
+                  <https://developers.italia.it/it/spid>
+
+              - label: "Qualcosa di utile per iniziare"
+                question: "Hai voglia di metterti in gioco?"
+                content:
+
+                    - label: "Contribuire sulla validazione dei test"
+                      content: |
+                       <https://github.com/italia/spid-sp-shibboleth/issues/9>
```

Comment:
I'm not sure we want this resolution and to link specific issues, they'll need to be maintained and removed from here when they are solved.

Maybe we can link to a GitHub search with specific labels? 

Wdyt @astagi @ruphy?

Reply:
@bfabio removed, don't mind